adserver/templates/adserver/reports/advertiser-geo.html

```diff
@@ -19,42 +17,22 @@
 {% endblock breadcrumbs %}
 
 
-{% block additional_filters %}
-{{ block.super }}
-
-<div class="col-xl-3 col-md-6 col-12 mb-3">
-  <label class="col-form-label" for="id_country">{% trans 'Country' %}</label>
-  <select class="form-control" name="country" id="id_country">
-    <option value="">{% trans 'All countries' %}</option>
-    {% for option, name in country_options %}
-      <option value="{{ option }}"{% if country == option %} selected{% endif %}>{{ name }}</option>
-    {% endfor %}
-  </select>
-</div>
-
-<div class="col-xl-3 col-md-6 col-12 mb-3">
-  <label class="col-form-label" for="id_flight">{% trans 'Flight' %}</label>
-  <select class="form-control" name="flight" id="id_flight">
-    <option value="">{% trans 'All flights' %}</option>
-    {% for flight in flights %}
-      <option value="{{ flight.slug }}"{% if flight.slug == request.GET.flight %} selected{% endif %}>{{ flight.name }}</option>
-    {% endfor %}
-  </select>
-</div>
-
-{% endblock additional_filters %}
-
 {% block explainer %}
 <section class="mb-5">
-  <h3>About this report</h3>
-  <p>
-    This report shows all your traffic that had country data we could determine from the IP address.
-  </p>
-  <em>
-  This report shows the <strong>top {{ limit }} geos</strong> and updates periodically. All previous days data is complete.
-  </em>
+  <h3>{% trans 'About this report' %}</h3>
+  <p>{% trans 'This report shows all your traffic that had country data we could determine from the IP address.' %}</p>
+  <em>{% trans 'All previous days data is complete. This report can take up to a minute to load.' %}</em>
```

Comment:
I'm not familiar with whether this is a singular previous day or plural previous days, but I'm assuming plural possessive is wanted here:

```suggestion
  <em>{% trans 'All previous days' data is complete. This report can take up to a minute to load.' %}</em>
```

Reply:
I can't take this suggestion directly as it causes a template syntax error with the translation. We could just do "All data for previous days is complete".

Reply:
Oops, missed the string quote style, yeah. I like your suggestion more either way.